CHANGELOG.rst

```diff
@@ -5,11 +5,13 @@ Changelog
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 
-0.3.0-SNAPSHOT (2021-07-20)
+0.3.0-SNAPSHOT (2021-08-18)
 ---------------------------
 
 **Added**
 
+* Download of available samples is possible as a manifest file (`#54 <https://github.com/qbicsoftware/sample-tracking-status-overview/pull/54/conflicts>`_)
```

Comment:
Is there a reason why this links to to the conflicts subsection of the pull request?